components/Collections/src/private/MDCCollectionViewEditor.m

```diff
@@ -344,11 +344,18 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
 
 - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
        shouldReceiveTouch:(UITouch *)touch {
-  if (self.isEditing) {
-    return YES;
-  } else {
-    return NO;
+  BOOL allowsSwipeToDismissItem = NO;
+  if ([_delegate respondsToSelector:@selector(collectionViewAllowsSwipeToDismissItem:)]) {
+    allowsSwipeToDismissItem = [_delegate collectionViewAllowsSwipeToDismissItem:_collectionView];
+  }
+
+  BOOL allowsSwipeToDismissSection = NO;
+  if ([_delegate respondsToSelector:@selector(collectionViewAllowsSwipeToDismissSection:)]) {
+    allowsSwipeToDismissSection =
+    [_delegate collectionViewAllowsSwipeToDismissSection:_collectionView];
```

Comment:
Hey @ajsecord looks like this got merged before I could fix. Fixing in new PR here: https://github.com/material-components/material-components-ios/pull/1428